plane/src/database/backend.rs

```diff
@@ -31,6 +31,33 @@ impl super::subscribe::NotificationPayload for BackendActionMessage {
     }
 }
 
+#[derive(Serialize, Deserialize, Debug, Clone)]
+pub struct BackendMetricsMessage {
+    pub backend_id: BackendName,
+    /// Memory used by backend excluding inactive file cache, same as use shown by docker stats
+    /// ref: https://github.com/docker/cli/blob/master/cli/command/container/stats_helpers.go#L227C45-L227C45
+    pub mem_used: u64,
+    /// Memory used by backend in bytes
+    /// (calculated using kernel memory used by cgroup + page cache memory used by cgroup)
+    pub mem_total: u64,
+    /// Active memory ( non reclaimable )
```

Comment:
nit: I know this code pre-existed, but since we're touching it, can we normalize the spacing on these comments?